proto/mgmt.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+// This protobuf is still in flux - please don't depend on it.
+
+package proto;
+
+message ListServersRequest {
+}
+
+message ListServersResponse {
+    message Server {
+        string family = 1;
+        string name = 2;
+    };
+    repeated Server server = 1;
+}
+
+message GetBGPServerStatusRequest {
+    string name = 1;
+}
+
+message GetBGPServerStatusResponse {
+    uint32 router_id = 1;
+    uint32 local_asn = 2;
+
+    message PeerInfo {
+        string peer_address = 1;
+        uint32 peer_asn = 2;
+        uint32 local_asn = 3;
+    };
+    repeated PeerInfo peers = 4;
+}
+
+service Management {
+    rpc ListServers(ListServersRequest) returns (ListServersResponse);
```

Comment:
What exactly is this RPC meant to be doing. I'm not sure what exactly a server is in this context. 
Maybe add a comment.